crypto/keyspb/keyspb.proto

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package keyspb;
+
+// Generator defines how a new pair of asymmetric keys should be generated.
+message Generator {
+  message ECDSA {
+    enum Curve {
+      P256 = 0;
```

Comment:
Suggestion: (Assuming you want P256 to be the default)

```proto
enum Curve {
  // Default curve size. Currently equivalent to P256.
  DEFAULT_CURVE = 0
  P224 = 1;
  P256 = 2;
  P384 = 3;
  P521 = 4;
}
```

Whereas, DEFAULT_CURVE is replaced by P256 server-side (for now). This allows us to increase the default without breaking the interface. (Note that responses should explicitly say P256 when DEFAULT_ comes in.)